tests/testBackground.py

```diff
@@ -513,22 +458,8 @@ def testOnlyOneGridCell(self):
                 testval = afwMath.cast_BackgroundMI(backobj).getPixel(bctrl.getInterpStyle(), xpix, ypix)
                 self.assertAlmostEqual(testval/mean, 1)
 
-    def testTicket1681OffByOne(self):
```

Comment:
If the test is explicitly checking whether something won't raise an exception, it should be wrapped like so:

```python
try:
    do_something()
except SomeError:
    self.fail("do_something() should not raise SomeError")
```

Otherwise, it's not clear what is being tested, if it fails it will be an "error" instead of a test "failure", and we can be clear about what exception we're checking for.

Since this test was disabled anyway, in an un-documented `if False`, we might as well delete it, since the test method was doing nothing.